cypress/e2e/templates.spec.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileLicenseText: 2023 Julius Härtl <jus@bitgrid.net>
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+describe('Create new office files from templates', function() {
+
+	let randUser
+	before(function() {
+		cy.createRandomUser().then(user => {
+			randUser = user
+			cy.login(user)
+		})
+	})
+	beforeEach(function() {
+		cy.login(randUser)
+		cy.createFolder(randUser, 'Templates-user')
+		cy.uploadFile(randUser, 'templates/presentation.otp', 'application/vnd.oasis.opendocument.presentation', '/Templates-user/presentation.otp')
+		cy.setPersonalTemplateFolder(randUser, '/Templates-user')
+		cy.login(randUser)
```

Comment:
Do you need to login again here?

Reply:
I guess it's because uploadFile will log you out. I think we have an uploadFile in text cypress that uses the current login and preserves it: https://github.com/nextcloud/text/blob/master/cypress/support/commands.js#L55-L76

Reply:
Ah thanks for the pointer, let me adapt to that the to avoid clearing.